politeiad/plugins/pi/pi.go

```diff
@@ -108,6 +112,10 @@ const (
 	// SettingProposalStartDateMin is the default minimum possible proposal
 	// start date - seconds from current time.
 	SettingProposalStartDateMin int64 = 604800 // One week in seconds.
+
+	// SettingAllowBillingStatusCorrection is the default value of the bool
+	// flag which determines whether billing status corrections are allowed.
+	SettingAllowBillingStatusCorrection = false
```

Comment:
This plugin setting needs to be more generic.  Something like a max allowed billing status changes that defaults to 1.  You don't know how the billing status implementation is going to change over time.  `Correction` implies details about the current implementation that could change.  

Also, `Correction` is too ambigious.  Would somebody reading this comment who is not familiar with the implementation know what it means?  Unlikely.

Reply:
Makes sense, I can try `maxAllowedBillingStatusChanges`.